tests/mgbench/workloads/pokec.py

```diff
@@ -300,6 +300,20 @@ def benchmark__basic__single_vertex_write_write(self):
             {"id": random.randint(1, self._num_vertices * 10)},
         )
 
+    def benchmark__basic__single_big_vertex_write_write(self):
+        return (
+            "CREATE (:UserTempBig {id : $id, p2: 42, "
+            'p3: "Here is some text that is not extremely short", '
+            'p4:"Short text", p5: 234.434, p6: 11.11, p7: false})',
+            {"id": random.randint(1, self._num_vertices * 10)},
+        )
+
+    def benchmark__basic__single_vertex_property_update_update(self):
```

Comment:
We have the same query below, this should be like the write above, but using `SET` for all the properties on the node that has been created. Like, having like: 
SET .. 
SET ..
SET ..